src/main.ts

```
@@ -57,7 +57,10 @@ export class OpenFoodFacts {
     fetch: typeof global.fetch,
     options: OpenFoodFactsOptions = { country: "world" },
   ) {
-    this.baseUrl = `https://${options.country}.openfoodfacts.org`;
+    this.baseUrl = options.host
+      ? `https://${options.host}`
+      : `https://${options.country}.openfoodfacts.org`;
+
```

1. Do not prefix the `options.host` url. We don't know what the user wants there.
2. I would just throw an exception if they are both given. It makes no sense.

---

Just pushed a fix. It no longer prefixes host, and throws an exception if both host and country are provided.